### PR TITLE
[TfL] Display more details when roadworks marker is selected

### PR DESCRIPTION
**Pull request by davea**
Shows details about selected roadworks asset in the sidebar:

image

For https://github.com/mysociety/fixmystreet-commercial/issues/1605

[skip changelog]

### CODE REVIEW

File: web/cobrands/tfl/assets.js [hunk 1]
```diff
@@ -266,8 +266,20 @@ var rw_stylemap = new OpenLayers.StyleMap({
     })
 });
 
+function format_date(date) {
```
Comment (dracos) [inline web/cobrands/tfl/assets.js]: Make the one in roadworks.js a function that can be called externally?

File: web/cobrands/tfl/assets.js [hunk 2]
```diff
@@ -312,7 +311,51 @@ fixmystreet.assets.add(fixmystreet.roadworks.layer_future, {
...
+            feature.layer = this;
+            var attr = feature.attributes,
+            tooltip = attr.tooltip.replace(/\\n/g, '\n'),
+            desc = attr.works_desc.replace(/\\n/g, '\n');
+
+            var $msg = $('<div class="js-roadworks-message js-roadworks-message-' + this.id + ' box-warning"></div>');
+            var $dl = $("<dl></dl>").appendTo($msg);
+            $dl.append("<dt>Location</dt>");
+            var $summary = $("<dd></dd>").appendTo($dl);
+            tooltip.split("\n").forEach(function(para) {
```
Comment (dracos) [inline web/cobrands/tfl/assets.js]: Probably not worth it, but feels quite a bit of this could be shared with the roadworks.js in some manner.

Comment (davea) [reply web/cobrands/tfl/assets.js]: Agreed on both counts. I did think about factoring it together somehow with more config options but decided not to bother.

File: web/cobrands/tfl/assets.js (hunk 1 above)
Comment (davea) [reply web/cobrands/tfl/assets.js]: Good idea, will do.

**Pull request merged**